neutronics/mcnp_inputs.py

```diff
@@ -0,0 +1,176 @@
+import math
+from pyne.material import Material, MaterialLibrary
+from string import Template
+import material_data as md
+
+
+def build_pyne_matlib(nucdata_file=None):
+    """Fetch pyne material from compendium.
+
+    This function builds PyNE material library
+    for UWNR non-fuel components defined in reactor_data.py
+
+    Arguments:
+        nucdata_file (str)[-]: Filename 'nuc_data.h5' with its full path.
+    """
+    h5path = "/home/alex/.local/lib/python3.5/site-packages/pyne/nuc_data.h5"
+    
+    if nucdata_file:
```

Comment:
Shouldn't this only be used if `nucdata_file is None`?

Reply:
I believe this behaves as I wanted it to. If the user supplies a path, it modifies the default path using the optional input.

Reply:
Is it ok to set the default for my machine? I did that for selfish (but I think sensible) reasons.